frontend/src/store/objectStore.ts

```diff
@@ -42,14 +43,14 @@ export const useObjectStore = defineStore('objectStore', () => {
     objectList.value = response.data;
   }
 
-  async function getObject(objectId: string) {
-    await objectService.getObject(objectId);
+  async function getObject(objectId: string, versionId?: string, download?: boolean) {
```

Comment:
not sure if you need the download parameter (?)

Reply:
Yea I agree - with the function signature change on service layer, that download parameter doesn't need to exist anymore.

Reply:
Indeed, only 1 get file mode at this point.